package.json

```diff
@@ -35,7 +35,7 @@
   },
   "dependencies": {
     "brace": "0.11.1",
-    "formik": "^2.2.5",
+    "formik": "^2.2.6",
```

Comment:
Instead of changing the version number in `package.json`, I suggest running `yarn upgrade` to update all the dependencies (including formik).
I think there are 2 reasons:
1. The symbol `^` stands for using the same major version as stated of the dependency, regardless the minor version. It will update `yanr.lock` file by picking the latest version of the dependency.
2. AD dashboards pluign is using version number of `^2.2.5` (https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/blob/main/package.json#L47), if we change it alone, with the dependency version consistency strategy of OpenSearch Dashboards, it will block people in building the 2 plugins together in same folder.